test/e2e/network_test.go

```diff
@@ -540,4 +540,50 @@ var _ = Describe("Podman network", func() {
 		nc.WaitWithDefaultTimeout()
 		Expect(nc.ExitCode()).To(Equal(0))
 	})
+
+	It("podman network prune", func() {
+		// Create two networks
+		// Check they are there
+		// Run a container on one of them
+		// Network Prune
+		// Check that one has been pruned, other remains
+		net := "macvlan" + stringid.GenerateNonCryptoID()
+		net1 := net + "1"
+		net2 := net + "2"
+		nc := podmanTest.Podman([]string{"network", "create", net1})
+		nc.WaitWithDefaultTimeout()
+		defer podmanTest.removeCNINetwork(net1)
+		Expect(nc.ExitCode()).To(Equal(0))
+
+		nc2 := podmanTest.Podman([]string{"network", "create", net2})
+		nc2.WaitWithDefaultTimeout()
+		defer podmanTest.removeCNINetwork(net2)
+		Expect(nc2.ExitCode()).To(Equal(0))
+
+		list := podmanTest.Podman([]string{"network", "ls", "--format", "'{{.Name}}'"})
```

Comment:
```suggestion
		list := podmanTest.Podman([]string{"network", "ls", "--format", "{{.Name}}"})
```
If you remove the single quotes here you don't have to add them below back in. `OutputToStringArray` does not add them

Reply:
fixed